Guide/src/reference/architecture/openhcl_servicing.md

```diff
@@ -0,0 +1,258 @@
+# OpenHCL Servicing
+
+## Overview
+
+OpenHCL servicing (also known as VTL2 servicing) is a mechanism that allows the host to update or service the OpenHCL paravisor while minimizing downtime and maintaining device state. During a servicing operation, OpenHCL can save its state, shut down, be updated by the host, and then restore its previous state when restarted.
+
+This capability is particularly important in production environments where maintaining continuous VM operation during paravisor updates is critical.
+
+## Servicing Lifecycle
+
+A servicing operation follows these steps:
+
+### 1. Servicing Request
+
+The host initiates a servicing operation by sending a request to OpenHCL with:
+- **Correlation ID**: For tracing and diagnostics
+- **Timeout Hint**: Deadline for completing the save operation
+- **Capabilities Flags**: Indicates what the host supports
+
+### 2. State Saving
```

Comment:
Reference the existing docs on save state here (and also link those docs to right here)